docs/guidesdraft/01-getting-started.md

```diff
@@ -0,0 +1,138 @@
+
+### Basic set up
+
+The easiest way to use CARTO VL is to include the required files from our CDN:
+
+* CARTO VL library: https://libs.cartocdn.com/carto-vl/%VERSION%/carto-vl.js
+* Minified file: https://libs.cartocdn.com/carto-vl/%VERSION%/carto-vl.min.js
+
+Then, we will need to add Mapbox GL JavaScript and CSS files.
+
+```html
+<head>
+  <!-- Include CARTO VL JS -->
+  <script src="https://libs.cartocdn.com/carto-vl/%VERSION%/carto-vl.min.js"></script>
+  <!-- Include Mapbox GL JS -->
+  <script src="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.js"></script>
+  <!-- Include Mapbox GL CSS -->
+  <link href="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.css" rel="stylesheet" />
+</head>
+```
+
+This will add the `carto` and the `mapboxgl` objects to the global namespace.
+
+> Note: Not every Mapbox GL version is compatible with CARTO VL. We highly recommend the following combination. If you are importing CARTO VL from npm, you have to use our Mapbox GL fork. Read more about how to do this in [the advanced guide]((https://carto.com/developers/carto-vl/guides/advanced))
+
+Create a `div` where the map is going to be drawn.
+
+```html
+  <div id="map"></div>
+```
+
+Style the `div` to ensure it will be displayed correctly.
+
+```css
+#map {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+}
+```
+
+#### Defining the basemap
+
+Once we have the `div`, we use the `mapboxgl` object to initialize our map using the following parameters:
+
+- **`container`** indicates where the map is going to be placed
+- **`style`** contains the information about the basemap
+- **`center`** indicates the area of the world we are going to visualize
+- **`zoom`** defines the default zoom level
+- **`dragRotate`** disables the map rotation
+
+```js
+const map = new mapboxgl.Map({
+      container: 'map',
+      style: 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
+      center: [0, 30],
+      zoom: 2,
+      dragRotate: false
+    });
+```
+
+For basemaps you can add [Mapbox custom styles](https://www.mapbox.com/mapbox-gl-js/style-spec/) or choose one of the three predefined styles offered by CARTO:
+
+- **Voyager:** https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json
+- **Positron:** https://basemaps.cartocdn.com/gl/positron-gl-style/style.json
+- **Dark Matter:** https://basemaps.cartocdn.com/gl/dark-matter-gl-style/style.json
+
+At this point you will have a basic map:
+
+[LiveExample](http://carto.com/developers/carto-vl/examples/maps/guides/getting-started/basic-set-up.html)
+
+### Defining the source
+
+In order to render data from CARTO you need to create a CARTO account and then get the necessary [credentials](https://carto.com/developers/fundamentals/authorization/).
+
+The first thing you need to do is [authenticate the client](https://carto.com/developers/carto-vl/reference/#cartosetdefaultauth) with your `user` and `apiKey`.
+
+```js
```

Comment:
I would explain something like, "For this guide, you can use *cartovl* account and the next credentials, that give you access to `populated_places` dataset " (I think most people will do that), ..."but you can easily change them with yours to visualize your own datasets"